editor/src/core/shared/element-template.ts

```diff
@@ -1641,6 +1641,7 @@ export interface ElementInstanceMetadata {
   attributeMetadatada: StyleAttributeMetadata | null
   label: string | null
   importInfo: ImportInfo | null
+  conditionalValue: boolean | null
```

Comment:
oh I had an idea! this type could be `boolean | 'not-a-conditional'`!

Reply:
https://github.com/concrete-utopia/utopia/pull/3388/commits/b76bf99eb2a5e23a3eccb599828d6d8389ddc2eb